src/app/components/EvaluationCard/EvaluationCard.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import styled from 'styled-components';
+import RoundActionButton from '../RoundActionButton/RoundActionButton';
+
+type CardProps = {
+  header: string;
+  name: string;
+  evaluation: string;
+  deleteCard: (name: string, evaluation: string) => void;
+};
+
+export default function Card({
+  header,
+  name,
+  evaluation,
+  deleteCard,
```

Comment:
I think it would make more sense to call this prop `onDeleteClick` or something that describes the event basically. You're not actually deleting a `Card` just some data that previously led to the `Card` being generated. Also, `deleteCard` describes what will happen, but from the Card's point of view, the actual functionality will be assigned later.